dist/languages/pl.json

```diff
@@ -0,0 +1,81 @@
+{
+  "Forien": {
+    "ChatCommanderWFRP4e": {
+      "Commands": {
+        "TableDescription": "Wybierz tablicę, z której chcesz wykonać rzut",
+        "TableHint": "Składnia: <span class='parameter'>klucz</span> <span class='parameter'>[kolumna]</span> <span class='parameter'>[modyfikator]</span>",
+        "TableHelp": "Pokaż wszystkie tablice jako interaktywne linki.",
+        "TableModExample": "Przykładowy modyfikator: <strong>{mod}</strong>",
+
+        "ConditionsDescription": "Wyślij opis Stanu na czat.",
+        "ConditionsHint": "Składnia: <span class='parameter'>nazwa</span>",
+
+        "PropertiesDescription": "Wyślij opis Zalety lub Wady przedmiotu na czat.",
+        "PropertiesHint": "Składnia: <span class='parameter'>nazwa</span>",
+
+        "CharGenDescription": "Otwórz Generator Postaci.",
+
+        "NameGenDescription": "Wygeneruj imię Postaci.",
+        "NameGenHint": "Składnia: <span class='parameter'>rasa</span> <span class='parameter'>[płeć]</span>",
+
+        "AvailabilityDescription": "Rzuć na Dostępność przedmiotu na podstawie wielkości osady i rzadkości.",
+        "AvailabilityHint": "Składnia: <span class='parameter'>osada</span> <span class='parameter'>dostępność</span>",
+
+        "PlayerPayDescription": "Zapłać określoną ilość pieniędzy z ekwipunku swojego Bohatera.",
+        "PlayerPayAmountHint": "Składnia: <span class='parameter'>ilość</span>",
+        "AmountExample": "Przykładowe ilości:",
+
+        "GMPayDescription": "Pozwala graczom na zapłacenie określonej kwoty lub automatycznie dokonuje transakcji.",
+        "GMPayAmountHint": "Składnia: <span class='parameter'>ilość</span> — wyświetla wiadomość pozwalającą na zapłatę dla wszystkich graczy",
+        "GMPayPlayerHint": "Składnia: <span class='parameter'>ilość</span> <span class='parameter'>player</span> — wyświetla wiadomość o zapłacie tylko dla jednego gracza",
+        "GMPayActorHint": "Składnia: <span class='parameter'>ilość</span> <span class='parameter'>actor</span> — automatycznie płaci postaci określoną <code>ilość</code> pieniędzy",
```

Comment:
Actor is paying the amount. It's not "amount is paid to character"

Reply:
Additionally you can change the `player` and `actor` parameters to localized text